types.go

```diff
@@ -1397,19 +1397,29 @@ func (t *ContractInterfaceType) InterfaceInitializers() [][]Parameter {
 
 // Function
 
+type FunctionPurity int
+
+const (
+	ImpureFunction = iota
+	ViewFunction   = 1
```

Comment:
Same as for the other enum:

```suggestion
	FunctionPurityUnspecified FunctionPurity = iota
	FunctionPurityView
```